cppcache/src/EntryExpiryHandler.cpp

```diff
@@ -56,25 +56,19 @@ int EntryExpiryHandler::handle_timeout(const ACE_Time_Value& current_time,
     }
 
     auto elapsed = curr_time - lastTimeForExp;
-    LOGDEBUG(
-        "Entered entry expiry task handler for key [%s] of region [%s]: "
-        "%s,%s,%s,%s",
-        Utils::nullSafeToString(key).c_str(),
-        m_regionPtr->getFullPath().c_str(),
-        to_string(curr_time.time_since_epoch()).c_str(),
```

Comment:
Removing the time components from this log message really makes the log message useless. Either fix the issue of printing the time on Windows or delete the log message entirely.

Reply:
Okay, will remove.  FWIW, it is/was working fine on Windows, the issue was that after I applied that fix it started breaking a test on Linux, and I couldn't figure out why in a reasonable amount of time.